core/sql/regress/qat/dqatdml04.known

```diff
@@ -1,32 +1,12 @@
-2571a2572
-> >>?ignore
-2576,2588d2576
-< 
-< A                     B
-< --------------------  ---------------------
-< 
-<                 2000                2100.00
-<                10000                2100.00
-<                16000                1300.00
-<                16000                2100.00
-<                18000                1600.00
-<                20000                 300.00
-<                20000                2100.00
-< 
-< --- 7 row(s) selected.
-2595,2607c2583
-< 
-< A                     B
-< --------------------  ---------------------
-< 
-<                 2000               2000.000
-<                10000               2000.000
-<                16000               1200.000
-<                16000               2000.000
-<                18000               1500.000
-<                20000                200.000
-<                20000               2000.000
-< 
+2454c2454
+<  100
```

Comment:
This looks like you changed the test also, removing the ?ignore, but the pull request doesn't show a change in the test file. Is that ok?